### ISSUE TITLE
Charis SIL might be worth giving a try

### ISSUE THREAD

**Issue by marespiaut** [koreader/koreader-fonts#3]
Hello,

As my title implies, the open font Charis SIL, based on Bitstream Charter, might be worth giving a try as one of the default KOReader fonts.

Best regards.

**Comment by tigran123** [comment]
There are so many nice fonts and just as many different tastes, that I doubt anyone would use the default font that comes in any software. I, for example, always change the default (in both Koreader and CoolReader) to something like Baskerville 10 Pro for most of my reading. But sometimes (especially for Russian books) I like to switch to the old good Academy or even the beautiful (especially for books with lots of text in italics) Adobe Garamond Premier Pro family.

I am NOT objecting to your suggestion. Btw, I tried Charis SIL and it looked "okay", but Baskerville 10 Pro is, imho, still better for English texts (but it is commercial, unfortunately).

**Comment by Frenzie** [comment]
There's this pseudo-Baskerville, btw.
http://www.tug.dk/FontCatalogue/librebaskerville/

I prefer Tex Gyre Pagella, a Palatino lookalike:
https://www.fontsquirrel.com/fonts/tex-gyre-pagella

**Comment by NiLuJe** [comment]
Yeah, the "default" font choice is made more in terms of unicode coverage than how nice it looks ;).

Hence Noto ;).

(Plus, I don't think we want to add yet another font to the mix, as fonts already take up far too much space in our packages ;)).

**Comment by pazos** [comment]
Speaking of unicode. I did notice that @baskerville updated a unicode test epub in plato. See https://github.com/baskerville/plato/blob/master/contrib/unicode_test.epub. 

I just played a bit with that file and how noto fonts worked with it. Not impressed. Searching in wikipedia for unicode coverage I found [Code2000](https://en.wikipedia.org/wiki/Code2000), which covers most of the glyphs. On the wiki page it says there's no official download site but can be downloaded from https://fonts2u.com/code2000.font. 

Are you aware of this font?. I think it is awesome and weights less than 3MB.

**Comment by NiLuJe** [comment]
Yep, license makes it unredistributable/unusable IIRC.

**Comment by NiLuJe** [comment]
Otherwise, there's [GNU Unifont](http://unifoundry.com/unifont/index.html), which is *really* not pretty in its TTF variant since it's a bitmap font at heart, but does aim for full coverage ;).
